### PR TITLE
updated photos, text, and attempted to fix header photo sizing issue

### PR DESCRIPTION
**Pull request by VictoriaFry**

### CODE REVIEW

**Comment by VictoriaFry** [comment]
@dmckinno  please approve my wonderful changes.

File: bigday/static/bigday/css/creative.css [hunk 1]
```diff
@@ -83,10 +83,15 @@ aside {
...
+/*!
+ *Victoria Fry adjusted the background color to transparent to not block dan's face
+ *If a different photo is available, better to change back to #fff
+ */
+
 .navbar-default {
     border-color: rgba(34,34,34,.05);
     font-family: 'Open Sans','Helvetica Neue',Arial,sans-serif;
-    background-color: #fff;
+    background-color: fff;
```
Comment (dmckinno) [inline bigday/static/bigday/css/creative.css:94]: Did you intend to delete this #?

File: bigday/templates/home.html [hunk 2]
```diff
@@ -45,7 +45,9 @@
     <header id="main-header">
         <div class="header-content">
             <div class="header-content-inner">
-                <h1>9.1.19</h1>
+
+                <!-- Inbetween <h1> and </h1> put the date to be displayed in front of photo-->
+                <h1>  </h1>
```
Comment (dmckinno) [inline bigday/templates/home.html:50]: You can just delete these heading tags.

**Review by dmckinno — APPROVED** [review]
Looks great!

File: bigday/static/bigday/css/creative.css (hunk 1 above)
Comment (VictoriaFry) [reply bigday/static/bigday/css/creative.css:94]: No, will fix on next commit.

File: bigday/templates/home.html (hunk 2 above)
Comment (VictoriaFry) [reply bigday/templates/home.html:50]: Wanted to leave it in just in case we want to add it later if we change pictures.

**Pull request merged**